integration/test/performancetest/resources/config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"agent": {
-		"metrics_collection_interval": 60,
+		"metrics_collection_interval": 1,
```

Comment:
Collecting at this interval means that we're getting 60x the number of metric values, and we're persisting all of the data into DynamoDB, right? I think it's fine - just pointing it out